.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -60,3 +60,8 @@ If applicable, please add screenshots to help explain your problem.
 
 **Additional context**
 Please feel free to add any other context about the problem here.
+
+________________________________________________________________
+**Disclaimer** 
+
+We appreciate your contribution whether it is a bug report, feature request, or pull request with improvement (hopefully). Please comply with the [Community ethics policy](https://opensource.guide/building-community/), and do not expect us to answer your requests immediately. Also, do not treat *GitHub issues* as a support channel. 
```

Comment:
The link is referencing a generic web page and not the Community ethics policy

Reply:
Fixed. Changed to a precise Acceptable Use document